services/audit-service/src/main/java/datawave/microservice/audit/controller/AuditController.java

```diff
@@ -141,13 +148,44 @@ private void sendMessage(AuditParameters parameters) {
     @RolesAllowed({"AuthorizedUser", "AuthorizedServer", "InternalUser", "Administrator"})
     @RequestMapping(path = "/audit", method = RequestMethod.POST)
     public String audit(@RequestParam MultiValueMap<String,String> parameters) {
-        if ((healthChecker != null && healthChecker.isHealthy()) || healthChecker == null) {
-            log.info("Received audit request with parameters {}", parameters);
-            restAuditParams.clear();
-            restAuditParams.validate(parameters);
-            sendMessage(restAuditParams);
+        log.info("Received audit request with parameters {}", parameters);
+        
+        restAuditParams.clear();
+        restAuditParams.validate(parameters);
+        
+        boolean success;
+        final long auditStartTime = System.currentTimeMillis();
+        long currentTime;
+        int attempts = 0;
+        
+        AuditProperties.Retry retry = auditProperties.getRetry();
```

Comment:
I don't think it will happen using normal spring boot property mapping, but AuditProperties has a setter for retry that allows a null retry to be set. Therefore, you must guard for that here, or throw an error in the setter if null is passed.

Reply:
Updated the setter to disallow null assignment.